doc/architecture-decisions/0008-controller-specs.txt

```diff
@@ -0,0 +1,40 @@
+# 1. Testing controllers
+
+Date: 2022-10-18
+
+1.
+"controller tests" have filenames like `spec/controllers/casa_admins_controller_spec.rb` and syntax like
```

Comment:
You mention it at the end of this file, but I think it would be great to mention directly in this section that controller specs are deprecated and they should only be used if a file exists, and preferably the whole file is rewritten and moved to the requests directory.

Reply:
I think it would also be great to open a ticket to have someone move controller specs to request specs (maybe one-at-a-time to avoid merge conflicts).

Reply:
agreed